doc/develop/west/sign.rst

```diff
@@ -100,8 +100,18 @@ The signing script used when running ``west flash`` can be extended or replaced
 to change features or introduce different signing mechanisms. By default with
 MCUboot enabled, signing is setup by the :file:`cmake/mcuboot.cmake` file in
 Zephyr which adds extra post build commands for generating the signed images.
-The file used for signing can be replaced by adjusting the ``SIGNING_SCRIPT``
-property on the `zephyr_property_target`, ideally done by a module using:
+The file used for signing can be replaced from a sysbuild scope (if being used)
+or from a zephyr/zephyr module scope, the priority of which is:
+
+* Sysbuild
+* Zephyr property
+* Default MCUboot script (if enabled)
+
+From sysbuild, ``-D<target>_SIGNING_SCRIPT`` can be used to set a signing script
+for a specific image or ``-DSIGNING_SCRIPT`` can be used to set a signing script
```

Comment:
This should probably include fuller `west build/cmake` command examples for sysbuild and vanilla, both of which can set `-DSIGNING_SCRIPT`.

Reply:
Fixed